### ISSUE TITLE
[Accessibility: Resize text] Unable to manage storage and interface on Template when zoom 200%

### ISSUE THREAD

**Issue by alinefm** [kimchi-project/kimchi#899]
The control buttons are hidden on the right page.

![2c2ad47b27843e0f](https://cloud.githubusercontent.com/assets/4921718/14020158/a774736a-f1b4-11e5-9dbe-854032e2b147.jpg)

![4b1329d177cd814d](https://cloud.githubusercontent.com/assets/4921718/14020164/ad755342-f1b4-11e5-978c-4406a2c18e32.jpg)
